src/graphql/client.ts

```diff
@@ -1,30 +1,47 @@
-import { createClient, cacheExchange, fetchExchange, subscriptionExchange } from 'urql';
-import { getEnvVar } from '@util/env';
+import {
+  createClient,
+  cacheExchange,
+  fetchExchange,
+  subscriptionExchange,
+  dedupExchange
+} from 'urql';
 import { SubscriptionClient } from 'subscriptions-transport-ws';
-
-const serverEndpoint = getEnvVar('CUSTOM_SERVER_ENDPOINT');
-const isDev = serverEndpoint?.includes('127.0.0.1');
-
-const socketScheme = isDev ? 'ws://' : 'wss://';
-const httpScheme = isDev ? 'http://' : 'https://';
-
-const WS_URL = `${socketScheme}${serverEndpoint}/api/graphql`;
+import { GRAPHQL_URL, WS_URL } from '@lib/api/url';
 
 const subClient = new SubscriptionClient(WS_URL, {
   reconnect: true,
   timeout: 10000,
   reconnectionAttempts: 3
 });
 
+export const getToken = () => {
+  let token: string = '';
+
+  try {
+    token = localStorage.getItem('token') ?? '';
```

Comment:
```suggestion
    return localStorage.getItem('token') ?? '';
```

With this you can remove the const and the return below the trycatch